apache/superset
# feat(api): Add option to enable sync on import

### SUMMARY
In order to bring the API more in line with the CLI, I have added the ability to specify `sync_columns` and `sync_metrics` when using the API.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
Set `sync_columns` or `sync_metrics` to true. The API should behave like the CLI when those flags are set.

### ADDITIONAL INFORMATION
